src/native/third_party/isa-l.gyp

```diff
@@ -125,47 +125,31 @@
             ],
             'sources': [
                 # include
-                'isa-l_crypto/include/md5_mb.h',
-                'isa-l_crypto/include/types.h',
-                'isa-l_crypto/include/multi_buffer.h',
                 'isa-l_crypto/md5_mb/md5_ctx_base.c',
```

Comment:
I'm not sure about if on x64 the base source should be *after* or *before* the other sources of sse/avx to allow them to take over when the CPU supports it, or the order doesn't matter. Can you check? Maybe use linux `nm` on the output library to see where the symbols come from...

Also I don't think it makes sense that we don't have any unit-test that will verify our build works for the target platform.